### DISCUSSION TITLE
Add functionality to skin editor for editing skin.ini in lazer

### DISCUSSION THREAD

**Discussion by TheFantasticLoki** [ppy/osu#17497] [Ideas]
I've seen a few skins that appear bugged due to something in the skin.ini file such as skins with dots instead of combo numbers on hitcircles displaying combo numbers on top of the dots.
We can always edit the skin.ini and then import again but we can only import and delete all skins at a time rather than removing 1 skin and reimporting it with a new skin.ini. This can be especially time consuming for players that have a lot of skins since it takes so long to import anything more than 10 skins at a time.
Giving us the ability to quickly edit the ini file from the built in skin editor would resolve the issue. It would also allow us to more easily tinker with our skins in game and see the resulting changes immediately.

**Comment by Theighlin** [comment]
Loosely related #5820

**Reply by bdach** [reply]
#12029 is more related here. probably going to need to use the same mechanism for manual edits.

**Comment by peppy** [comment]
Aside, the `skin.ini` will never be editable in-game. But gradually the settings that were once in that file will be exposed via the skin editor, on individual components.

Until then you should edit as you used to then re-import the skin.
